src/seedu/addressbook/AddressBook.java

```diff
@@ -371,6 +380,8 @@ private static String executeCommand(String userInputString) {
         switch (commandType) {
         case COMMAND_ADD_WORD:
             return executeAddPerson(commandArgs);
+        case COMMAND_EDIT_WORD:
+            return executeEditPerson(commandArgs);   //newly add command
```

Comment:
This comment seems unnecessary in the long run, especially when more commands are added.